package.json

```diff
@@ -14,9 +14,9 @@
     "build": "lerna run build --stream",
     "test": "npm run build && npm run test:unit",
     "test:integration": "lerna run test:integration --stream",
-    "test:integration:windows": "lerna run test:integration --stream",
+    "test:integration:windows": "lerna run test:integration --stream --ignore polymer-linter --ignore web-component-tester",
```

Comment:
linter's integration test script was removed, so we don't need this `--ignore polymer-linter`

Reply:
Cool.   Removed!